app/views/friendships/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class = "summoner-friends" id="summoner-friends-list">
+  <div class = "col-md-5 col-md-offset-3">
+    <div class="table-responsive">
+    	<table class="table friends-list-table" cellspacing="0" style="border-spacing: 0;">
```

Comment:
Get rid of this styling in your html.  Also, never use cellspacing="0".  `border-collapse: collapse;` does the exact same thing